Add docs for automatic motor direction detection (#91)
bluerobotics/ardusub-gitbook: getting-started/initial-setup.md

## Configuring Motor Directions

### Automatic Motor Direction Setup

As of ArduSub 4.0, it is possible to automatically detect the thrusters directions and reverse them if necessary.

> To do so, put the ROV in the water away obstacles, and click "Auto-Detect Directions".

Reviewer: ```suggestion
To do so, put the ROV in the water away from obstacles, and click "Auto-Detect Directions".
```

> The Rov will briefly pulse each thruster and check the IMU outputs for the desired outputs, if the output is reversed, it reverses the motor.

Reviewer: ```suggestion
The ROV will briefly pulse each thruster and check if the IMU output matches the desired vector orientation, if doesn't the motor will be reversed.
```
